plugin/src/main/java/net/dzikoysk/funnyguilds/config/range/IntegerRange.java

```diff
@@ -20,16 +20,18 @@ public IntegerRange(int minRange, int maxRange) {
         this.maxRange = maxRange;
     }
 
-    public int getMinRange() {
+    @Override
+    public Integer getMinRange() {
         return this.minRange;
     }
 
-    public int getMaxRange() {
+    @Override
+    public Integer getMaxRange() {
         return this.maxRange;
     }
 
     public static <V> Option<V> inRange(int value, Map<IntegerRange, V> rangeMap) {
-        for (Entry<IntegerRange, V> entry : rangeMap.entrySet()) {
+        for (Map.Entry<IntegerRange, V> entry : rangeMap.entrySet()) {
```

Comment:
Czy to `Map.` jest wymagane przez checkstyle?

Reply:
Nie, chyba po prostu się przypadkiem dodał